todoman/model.py

```diff
@@ -603,7 +605,13 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             max_due = (datetime.now() + timedelta(hours=due)).timestamp()
             extra_where.append('AND due IS NOT NULL AND due < ?')
             params.append(max_due)
-
+        if start:
```

Comment:
Instead of indexing you can write `is_before, dt = start` here and use those variables. `is_before` is then the same as `start[0]`, and `dt` is the same as `start[1]`. This also validates the length of the list at the same time.